pkg/virt-handler/vm.go

```diff
@@ -2660,10 +2660,7 @@ func (d *VirtualMachineController) updateFunc(_, new interface{}) {
 func (d *VirtualMachineController) addDomainFunc(obj interface{}) {
 	domain := obj.(*api.Domain)
 	log.Log.Object(domain).Infof("Domain is in state %s reason %s", domain.Status.Status, domain.Status.Reason)
-	key, err := controller.KeyFunc(obj)
-	if err == nil {
-		d.Queue.Add(key)
-	}
+	d.addFunc(obj)
```

Comment:
we don't want a change to the domain to indicate that the VMI informer has been updated (lower expectations). So this was also correct the previous way where `d.addFunc` was not used.